packages/table/src/cell/editableCell.tsx

```diff
@@ -40,45 +41,81 @@ export interface IEditableCellProps extends ICellProps {
     onConfirm?: (value: string) => void;
 }
 
-export class EditableCell extends React.Component<IEditableCellProps, {}> {
-    private cellElement: HTMLElement;
+export interface IEditableCellState {
+    /**
+     * Stores the editing state of the cell
+     */
+    isEditing: boolean;
+}
+
+@PureRender
+export class EditableCell extends React.Component<IEditableCellProps, IEditableCellState> {
+    public constructor(props: IEditableCellProps, context?: any) {
+        super(props, context);
+        this.state = {
+            isEditing: false,
+        };
```

Comment:
preferable to do the simple `public state = { ... }` instead of defining the whole constructor (unless of course you need the constructor but pretty sure you don't here)